man/systemd.netdev.xml

```diff
@@ -343,8 +343,16 @@
             </para>
           </listitem>
         </varlistentry>
+          <varlistentry>
+          <term><varname>EnableSTP=</varname></term>
+          <listitem>
+            <para>A boolean. This setting controls the IFLA_BR_STP_STATE option in the kernel.
```

Comment:
Please don't mention the internal netlink option for this functionality. A C identifier is hardly something we should confuse users wiht.